src/lib/logger.js

```diff
@@ -19,28 +19,23 @@
 *
 */
 
-import { DefaultLogger } from './DefaultLogger';
-import { ILogger } from '../types';
+import bunyan from 'bunyan';
+import { ILoggerCreate, ILogger } from '../types';
+import path from 'path';
 
-export default class Logger {
-  static _logger: ILogger = DefaultLogger;
-
-  static error(...params: Array<any>) {
-    Logger._logger.error(...params);
-  }
-
-  static info(...params: Array<any>) {
-    Logger._logger.info(...params);
-  }
-
-  static initialize(logger: ILogger) {
-    Logger._logger = logger;
-  }
-
-  static log(...params: Array<any>) {
-    Logger._logger.log(...params);
+export default class Logger implements ILoggerCreate {
+  static createLogger(applicationName: string): ILogger {
+    return bunyan.createLogger({
+      level: process.env.LOG_LEVEL !== undefined ? process.env.LOG_LEVEL : 'info',
```

Comment:
this can be simplified to `process.env.LOG_LEVEL || 'info'`
But all our config now in settings.js so you should add LOG_LEVEL to settings and do
 `process.env.LOG_LEVEL || settings.LOG_LEVEL`